src/main/java/uk/gov/digital/ho/hocs/workflow/api/WorkflowService.java

```diff
@@ -131,10 +153,17 @@ public GetCaseDetailsResponse getReadOnlyCaseDetails(UUID caseUUID) {
         GetCaseworkCaseDataResponse inputResponse = caseworkClient.getFullCase(caseUUID);
```

Comment:
I feel like this call could have been pushed up and made to contain the list of 'visited' stages (in order). Especially with the 'full' flag being passed in which we could use to drive the logic around when should we include the list of stage names.

Reply:
Just to make things clear, this is just advice on how we could have done it slightly neater, no need to action this item at the moment:)